include/obake/series.hpp

```diff
@@ -3882,8 +3883,25 @@ struct series_default_degree_impl {
             assert(ss != nullptr);
 
             if constexpr (al == 1) {
+#if !defined(_MSC_VER) || defined(__clang__)
```

Comment:
@7ofNine I had to enable a workaround for MSVC here as it would refuse to compile the following (legal) code on my CI setup. Would you mind checking if the workaround is necessary on the latest MSVC version as well? In such a case, I could use the ``OBAKE_MSVC_SUPPORTED`` macro here instead...